### DISCUSSION TITLE
silero_vad.jit can not loaded by torch correctly

### DISCUSSION THREAD

**Discussion by lizezheng** [snakers4/silero-vad#298] [Q&A] [answered]
## To Reproduce

import torch
model = torch.load('silero_vad.jit')

**Comment by lizezheng** [comment]
@zh794390558

**Comment by snakers4** [comment, marked as answer]
This is not a plain torch model.
Please see
https://github.com/snakers4/silero-vad/blob/4f57fae3fa68c456758dc64c01ee7ccaf507b5ac/utils_vad.py#L146
